ctr/main.go

```diff
@@ -50,5 +63,5 @@ func main() {
 
 func fatal(err string, code int) {
 	fmt.Fprintf(os.Stderr, "[ctr] %s\n", err)
-	os.Exit(code)
+	panic(Exit{code})
```

Comment:
This is mostly so the `restoreAndCloseStdin()` defer gets correctly invoked to reset the terminal state.